src/lib/utils/thorchain/memo.ts

```diff
@@ -71,6 +95,8 @@ const assertIsValidBasisPoints = (basisPoints: string, memo: string) => {
 export const assertAndProcessMemo = (memo: string): string => {
   const [action] = memo.split(':')
 
+  if (!assertMemoHasAction(action, memo)) throw new Error(`action is required in memo: ${memo}`)
```

Comment:
And same here, the right side will never evaluate, we could instead return an empty string and add a similar comment to the others?

Reply:
c437ee2da686cdb40220e4ec464f2dd3093eee20